apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -24,14 +27,52 @@ const styles: Styles<Theme, object> = {
     },
 };
 
+const columnList: { value: string; label: string }[] = [
+    { value: 'sectionCode', label: 'Code' },
+    { value: 'sectionDetails', label: 'Type' },
+    { value: 'instructors', label: 'Instructors' },
+    { value: 'dayAndTime', label: 'Times' },
+    { value: 'location', label: 'Places' },
+    { value: 'sectionEnrollment', label: 'Enrollment' },
+    { value: 'restrictions', label: 'Restrictions' },
+    { value: 'status', label: 'Status' },
+];
+
 interface CoursePaneButtonRowProps {
     classes: ClassNameMap;
     showSearch: boolean;
+    columnSelector: boolean;
     onDismissSearchResults: () => void;
     onRefreshSearch: () => void;
 }
 
 class CoursePaneButtonRow extends PureComponent<CoursePaneButtonRowProps> {
+    state = {
+        columnSelector: false,
+        activeColumns: [
```

Comment:
I would make column names an enum and make this an enum array so that all the naming is consistent. You can then translate it in `handleChange`. Alternatively, you can have an interface of `Record<ColumnName, Boolean>`.

Also, I don't get why the column names can't be the name that is displayed

Reply:
I mentioned this on Disc, but lemme try to reword it a bit better. I'm not entirely sure how to implement this because my current logic is as follows:

1. `CoursePaneButtonRow` is updated, emitting the new set of activeColumns to `SectionTable` & `SectionTableBody`. 
2. Both SectionTables then translate the activeColumns based on what they're rendering (Header strings vs Components)

Because there's slight variations, (CPBR has Restrictions but ST has Rstr, STB is components not strings), I'm not sure how to transfer the states through RightPaneStore without an in-between through value

<img width="1566" alt="Screenshot 2023-06-29 at 10 13 44 PM" src="https://github.com/icssc/AntAlmanac/assets/100006999/40b30ac2-050f-4f2c-b191-adc426c7280a">